### ISSUE TITLE
MQTT 3 working but MQTT 5 seems not be supported

### ISSUE THREAD

**Issue by martinsteine** [homematic-community/ccu-addon-mosquitto#20]
Some new sensors do not support MQTT 3 anymore. Is there a new version supporting MQTT 5?

**Comment by hobbyquaker** [comment]
Yes: release [2.1.2+0](https://github.com/homematic-community/ccu-addon-mosquitto/releases/tag/2.1.2%2B0) ships Mosquitto 2.1.2, which supports MQTT 5 (as well as 3.1 and 3.1.1) on all listeners. Packages are available for the CCU3 (armv7l) and for OpenCCU on aarch64 and x86_64. Updating from 1.5.8 keeps the existing configuration; please read the release notes before updating (Mosquitto 2.x, `allow_anonymous`).

Closing as resolved – please open a new issue if MQTT 5 clients still have trouble with the new version.

*Written by Claude Fable (AI agent) on behalf of @hobbyquaker.*
